### PR TITLE
css fixes for ported components

### PR DESCRIPTION
**Pull request by iburley33**

### CODE REVIEW

File: ib_client_frontend/src/components/leftcard/leftcardcss.css [hunk 1]
```diff
@@ -1,87 +1,84 @@
...
-    height: 25%;
-    object-fit: fill;
-    margin-bottom: 20px;
-  }
-  a {
-    text-decoration: none;
-  }
-  
\ No newline at end of file
+  --break-line: #BA2D0B;
```
Comment (mrogers23458) [inline ib_client_frontend/src/components/leftcard/leftcardcss.css:2]: root variables can be moved to app.css and then accessed in any child component that has it's own co-located css file. I would move these out put them app.css

File: ib_client_frontend/src/components/topcard/TopCard.jsx [hunk 2]
```diff
@@ -8,14 +8,18 @@ function Topcard({ image, title, description, link, detail }) {
 
     <div className='topCard'>
       <div className='topImgDiv'>
-          <img src={image} class='topImg'></img>
+        <img src={image} class='topImg'></img>
```
Comment (mrogers23458) [inline ib_client_frontend/src/components/topcard/TopCard.jsx:11]: missing alt tag, bad for seo. Add alt tag.

File: ib_client_frontend/src/components/topcard/topcardcss.css [hunk 3]
```diff
@@ -1,85 +1,88 @@
...
-    flex-direction: column;
-    justify-content: space-around;
-    max-height: 30%;
-    width: 100%;
-    height: 25%;
-    object-fit: fill;
-    margin-bottom: 20px;
-  }
+  --break-line: #BA2D0B;
+  --head-txt: #8c8c8a;
```
Comment (mrogers23458) [inline ib_client_frontend/src/components/topcard/topcardcss.css:3]: if you move all root variables to app.css code will be more dry.

File: ib_client_frontend/src/components/topcard/TopCard.jsx [hunk 4]
```diff
@@ -8,14 +8,18 @@ function Topcard({ image, title, description, link, detail }) {
...
       </div>
       <div className='topContent'>
-        <p class="pTop">{title}</p>
-        <a href={link} class='topLink'>
-          <h1 class='topDesc'>{description}</h1>
-        </a>
-        <h5 class='topDetail'>{detail}</h5>
+        <div className='topDescDiv'>
+          <p class="pTop">{title}</p>
+          <a href={link} class='topLink'>
```
Comment (mrogers23458) [inline ib_client_frontend/src/components/topcard/TopCard.jsx:16]: Probably shouldn't wrap an h1 in a href, can just use the h1, add an onclick and use useNavigate from react-router-dom.

File: ib_client_frontend/src/components/topcard/topcardcss.css (hunk 3 above)
Comment (iburley33) [reply ib_client_frontend/src/components/topcard/topcardcss.css:3]: complete

**Pull request merged**